src/main/resources/assets/farmersdelight/lang/lzh.json

```
@@ -0,0 +1,322 @@
+{
+  "block.farmersdelight.stove": "灶",

```

Prefer using `竈`, because it is the older Chinese glyph of `灶`.
建議使用`灶`的舊字形`竈`。
```suggestion
  "block.farmersdelight.stove": "竈",
```